src/Equinor.ProCoSys.IPO.Command/InvitationCommands/IInvitedPersonForCommand.cs

```diff
@@ -4,8 +4,7 @@ namespace Equinor.ProCoSys.IPO.Command.InvitationCommands
 {
     public interface IInvitedPersonForCommand
     {
-        Guid? AzureOid { get; }
-        string Email { get; }
+        Guid AzureOid { get; }
```

Comment:
Check line 203 and 206 in EditInvitationCommandHandler. Checks for AzureOid is null